crates/explorer/src/parse/token.rs

```diff
@@ -398,44 +398,6 @@ pub fn parse_token(
                 }),
             })
         }
-        TokenInstruction::GetAccountDataSize => {
-            check_num_token_accounts(&instruction.accounts, 1)?;
-            Ok(ParsedInstructionEnum {
-                instruction_type: "getAccountDataSize".to_string(),
-                info: json!({
-                    "account": account_keys[instruction.accounts[0] as usize].to_string(),
-                }),
-            })
-        }
-        TokenInstruction::InitializeImmutableOwner => {
-            check_num_token_accounts(&instruction.accounts, 1)?;
-            Ok(ParsedInstructionEnum {
-                instruction_type: "initializeImmutableOwner".to_string(),
-                info: json!({
-                    "account": account_keys[instruction.accounts[0] as usize].to_string(),
-                }),
-            })
-        }
-        TokenInstruction::AmountToUiAmount { amount } => {
-            check_num_token_accounts(&instruction.accounts, 1)?;
-            Ok(ParsedInstructionEnum {
-                instruction_type: "amountToUiAmount".to_string(),
-                info: json!({
-                    "account": account_keys[instruction.accounts[0] as usize].to_string(),
-                    "amount": amount.to_string(),
-                }),
-            })
-        }
-        TokenInstruction::UiAmountToAmount { ui_amount } => {
-            check_num_token_accounts(&instruction.accounts, 1)?;
-            Ok(ParsedInstructionEnum {
-                instruction_type: "uiAmountToAmount".to_string(),
-                info: json!({
-                    "account": account_keys[instruction.accounts[0] as usize].to_string(),
-                    "uiAmount": ui_amount.to_string(),
-                }),
-            })
-        }
```

Comment:
1. This is removed because the the enum no longer has these variants, right?
2. Is it because of downgrade of `solana-sdk`?
3. We are bringing it back probably, when we update it some time in the future, right?

Reply:
Yes, you are right. Once we will update we can bring it back.